### ISSUE TITLE
Health check path not available

### ISSUE THREAD

**Issue by herzo175** [cn-terraform/terraform-aws-ecs-fargate-service#4]
Hello!

I ran into this issue while trying to override the health check path:

```
Error: Unsupported argument

  on deployment/service.tf line 184, in module "ecs-fargate-service":
 184:   health_check_path   = "/healthcheck"

An argument named "health_check_path" is not expected here.
```

It looks like this variable isn't available but terraform documentation shows it here: https://registry.terraform.io/modules/cn-terraform/ecs-fargate-service/aws/1.0.6. Can you look into this please for overriding the path?

Thank you so much for reviewing this request and your work on this project.

**Comment by jnonino** [comment]
Hi, how are you?
Sorry for the delay, I was on holiday.
It should be available in version 1.0.7.
let me know if you have any comments

**Comment by herzo175** [comment]
Thank you, hope you had a great holiday!
